.golangci.yaml

```
@@ -11,17 +13,20 @@ linters:
+        - "all"
 
-linters-settings: # please keep this alphabetized
-  misspell:
-    locale: US
-  staticcheck:
-    checks:
-      - "all"
+formatters:
+  disable-all: true
+  enable:
+    - gofmt
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)